paasta_tools/chronos_tools.py

```diff
@@ -191,6 +192,12 @@ def read_chronos_jobs_for_service(service, cluster, soa_dir=DEFAULT_SOA_DIR):
 
 
 def load_chronos_job_config(service, instance, cluster, load_deployments=True, soa_dir=DEFAULT_SOA_DIR):
+    log.info("Reading general configuration file: service.yaml")
+    general_config = service_configuration_lib.read_service_configuration(
```

Comment:
I wish we had a more universal place to put this and then dedupe code that does the same thing for chronos/marathon/adhoc/native/etc.

Suggestions welcome on better architecting this code so you won't have to copy paste this a bunch of times :)